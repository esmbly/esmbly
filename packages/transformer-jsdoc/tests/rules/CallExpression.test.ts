import testRunner from '../helpers/stringTestRunner';

describe('rule: CallExpression', () => {
  it('adds a type argument to the call expression', async () => {
    const program = `
      /**
       * @type {i32}
       * @typeArgument
       */
      someFn(1)
    `;
    const [{ content }] = await testRunner(program);
    expect(content).toMatchSnapshot();
  });
});
