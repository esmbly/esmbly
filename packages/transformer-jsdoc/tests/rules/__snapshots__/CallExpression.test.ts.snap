// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rule: CallExpression adds a type argument to the call expression 1`] = `
"
      /**
       * @type {i32}
       * @typeArgument
       */
      someFn<i32>(1)
    "
`;
